src/github.com/UKHomeOffice/s3secrets/main.go

```diff
@@ -36,17 +38,15 @@ func main() {
 		fmt.Printf("s3secrets %s\n", Version)
 		os.Exit(0)
 	}
-	if bucket == "" {
-		fmt.Println("Please specify S3 bucket name. Exiting..")
-		os.Exit(1)
-	}
-	if !dirExists(outputDir) {
-		fmt.Println("Output dir does not exist. Exiting..")
+
+	if bucket == "" || !dirExists(outputDir) {
+		flag.Usage()
 		os.Exit(1)
 	}
+
 	var cfg *aws.Config
 	if region != "" {
-		cfg = &aws.Config{Region: region}
+		cfg = &aws.Config{Region: &region}
```

Comment:
Weird, if I don't do that it doesn't build... how do you build it?

I'm using go 1.5.1